ingestion/src/metadata/ingestion/source/dashboard/dashboard_service.py

```diff
@@ -127,6 +127,7 @@ class DashboardServiceTopology(ServiceTopology):
                 context="dataModel",
                 processor="yield_bulk_datamodel",
                 consumer=["dashboard_service"],
+                ack_sink=False,
```

Comment:
We removed the `ack_sink` from the Topology model to simplify this. Since here we don't want to run the GET in the topology_runner, we can just remove the `context` and manage the context in the `source` itself

Reply:
AFAIK, the default value of `ack_sink=True`, so if we remove `ack_sink` from the Topology model, it can cause previous error which have been describe in here: https://github.com/open-metadata/OpenMetadata/issues/13067

Reply:
just remove the context as well and it won't be added. You are controlling the context on the source itself

Reply:
Removed `context` and `ack_sink` params